lib/experimental/interactivity-api/directives/wp-context.php

```diff
@@ -24,7 +24,10 @@ function gutenberg_interactivity_process_wp_context( $tags, $context ) {
 	}
 
 	$new_context = json_decode( $value, true );
-	// TODO: Error handling.
+	if ( null === $new_context ) {
+		// Invalid JSON defined in the directive.
+		return;
+	}
```

Comment:
In the future, this type of error handling should be centralized in the `WP_Directive_Context` class so anyone using it would benefit.

Actually, it's pretty basic, but there is already protection against falsy values like `null` in the `set_context` method: [link](https://github.com/WordPress/gutenberg/blob/06620e9870ae33381a0005e2e49c8eb589670e90/lib/experimental/interactivity-api/class-wp-directive-context.php#L63-L67)